build.csx

```diff
@@ -153,7 +153,7 @@ targets.Add(
 
 targets.Add(
     "pack",
-    DependsOn("build", "outputDirectory"),
+    DependsOn("build", "pdbgit"),

```

Comment:
Why not just add `outputDirectory` as a dependency of `pack`? It's the only target that uses this directory.